package.json

```diff
@@ -3,17 +3,18 @@
   "version": "2.2.0",
   "description": "Helps you to release TAO extensions",
   "main": "index.js",
+  "engines" : { "node" : ">=14.0.0" },
   "scripts": {
     "test:dev": "nodemon -e js --watch tests --watch src --delay 1 --exec npm run test",
     "test:cov": "nyc npx tape \"tests/**/test.js\" ",
     "test:unit:cov": "nyc npm run test:unit",
     "test:cov:html": "nyc report --reporter=lcov && open-cli coverage/lcov-report/index.html",
     "test:cov:clover": "nyc report --reporter=clover",
-    "test": "tape \"tests/**/test.js\" | tap-diff || true",
-    "test:unit": "tape \"tests/unit/**/test.js\" | tap-diff || true",
-    "test:integ": "tape \"tests/integration/**/test.js\" | tap-diff || true",
+    "test": "tape \"tests/**/test.js\" | node_modules/.bin/tap-spec || true",
+    "test:unit": "tape \"tests/unit/**/test.js\" | node_modules/.bin/tap-spec || true",
+    "test:integ": "tape \"tests/integration/**/test.js\" | node_modules/.bin/tap-spec || true",
```

Comment:
**chore:** Is it needed to specify the path? Wouldn't `npx` does the trick?

**suggestion:** There is no need to specify the path for internal tooling
```suggestion
    "test": "tape \"tests/**/test.js\" | tap-spec || true",
    "test:unit": "tape \"tests/unit/**/test.js\" | tap-spec || true",
    "test:integ": "tape \"tests/integration/**/test.js\" | tap-spec || true",
```